tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -193,6 +193,10 @@ static cl::opt<bool>
     SPIRVMemToReg("spirv-mem2reg", cl::init(false),
                   cl::desc("LLVM/SPIR-V translation enable mem2reg"));
 
+static cl::opt<bool> SPIRVPreserveAllFunctionAttributesAndMetadata(
+    "spirv-preserve-all-function-attributes-and-metadata", cl::init(false),
```

Comment:
This is a very long option. How about something shorter like `--spirv-preserve-auxdata`?

Reply:
sure, thanks